pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -416,50 +424,50 @@ def test_aggregate_subsampling_count_cells():
     pd.testing.assert_frame_equal(count_df, expected_count, check_names=False)
 
 
-def test_aggregate_subsampling_profile():
-    result = ap_subsample.aggregate_profiles()
-
-    expected_subset = pd.DataFrame(
-        {
-            "TableNumber": sorted(["x_hash", "y_hash"] * 2),
-            "ImageNumber": sorted(["x", "y"] * 2),
-            "Metadata_Plate": ["plate"] * 4,
-            "Metadata_Well": sorted(["A01", "A02"] * 2),
-            "Metadata_ObjectNumber": [46, 3] * 2,
-        }
-    )
-
-    pd.testing.assert_frame_equal(ap_subsample.subset_data_df, expected_subset)
-
-
-def test_aggregate_subsampling_profile_compress():
-    compress_file = os.path.join(tmpdir, "test_aggregate_compress.csv.gz")
-
-    _ = ap_subsample.aggregate_profiles(
-        output_file=compress_file, compression_options={"method": "gzip"}
-    )
-    result = pd.read_csv(compress_file)
-
-    expected_result = pd.DataFrame(
-        {
-            "Metadata_Plate": ["plate", "plate"],
-            "Metadata_Well": ["A01", "A02"],
-            "Cells_a": [110.0, 680.5],
-            "Cells_b": [340.5, 201.5],
-            "Cells_c": [285.0, 481.0],
-            "Cells_d": [352.0, 549.0],
-            "Cytoplasm_a": [407.5, 705.5],
-            "Cytoplasm_b": [650.0, 439.5],
-            "Cytoplasm_c": [243.5, 78.5],
-            "Cytoplasm_d": [762.5, 625.0],
-            "Nuclei_a": [683.5, 171.0],
-            "Nuclei_b": [50.5, 625.0],
-            "Nuclei_c": [431.0, 483.0],
-            "Nuclei_d": [519.0, 286.5],
-        }
-    )
-
-    pd.testing.assert_frame_equal(result, expected_result)
+# def test_aggregate_subsampling_profile():
```

Comment:
I'm going to look into this now, separately from this review. I'll see if I can figure out what's going on